<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_ECM</class>
 <widget class="QMainWindow" name="MainWindow_ECM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView_ecm"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_xml_open"/>
    <addaction name="action_xml_save"/>
    <addaction name="separator"/>
    <addaction name="actiont_exit"/>
   </widget>
   <widget class="QMenu" name="menu_windows">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="action_list"/>
    <addaction name="action_toolBar"/>
   </widget>
   <widget class="QMenu" name="menu_build">
    <property name="title">
     <string>构建</string>
    </property>
    <addaction name="action_run"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_windows"/>
   <addaction name="menu_build"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_run_2"/>
   <addaction name="action_xml_open_2"/>
   <addaction name="action_xml_save_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>电子对抗组件列表</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>7</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QToolBox" name="toolBox">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <property name="tabSpacing">
          <number>6</number>
         </property>
         <widget class="QWidget" name="page_arithmetic">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>256</width>
            <height>406</height>
           </rect>
          </property>
          <attribute name="label">
           <string>算法组件</string>
          </attribute>
          <attribute name="toolTip">
           <string>放置算法组件的说明文字</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="5">
            <widget class="QListWidget" name="listWidget_arithmetic">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_background">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>256</width>
            <height>406</height>
           </rect>
          </property>
          <attribute name="label">
           <string>背  景</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QListWidget" name="listWidget_background">
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_list">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/property.png</normaloff>:/img/property.png</iconset>
   </property>
   <property name="text">
    <string>组件列表</string>
   </property>
  </action>
  <action name="action_xml_open">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="text">
    <string>打开xml文件</string>
   </property>
  </action>
  <action name="action_xml_save">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存为xml文件</string>
   </property>
  </action>
  <action name="action_run">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/runXml.png</normaloff>:/img/runXml.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="actiont_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_run_2">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/runXml.png</normaloff>:/img/runXml.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="action_xml_open_2">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_xml_save_2">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_toolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Res/res.qrc"/>
 </resources>
 <connections/>
</ui>
